docs/deployment/microbit-deployment-draft.md

```diff
@@ -0,0 +1,166 @@
+
+### Storing the config for each deployment
+
+The config for any deployment should be stored alongside that deployment so that it can be queried by other versions.
+
+### Locations for named versions:
+
+These versions will live at `python.microbit.org/<name>/v<version>`
+
+The top level link at `<branchname>` will redirect to the latest version, and the `<version>` redirects will work exactly as in the spec above.
+
+## Workflows for telling people about newer versions.
+
+When a user arrives at any version of the editor, the editor will check for the existence of a version manifest. If found the editor can decide what information to present to the user.
+
+If the editor is not the latest version AND the user has not already been redirected from a newer version (For example with ?redirect-from= maybe) then they will be prompted to move to the newest version. The text of the users’s script should be preserved as they user moves to the new version.
+In arriving at the newer version, the user should be told that they have arrived from an older version, and a means by which they can return to the version that the originally come to (presumably the version that the script they were loading was created in, or at least last published from) in case things don’t work
+
+ * Ideally, we should maintain a growing list of API breakages in newer versions of the editor as they happen so that we can scan code and warn users if it uses an API that has changed. 
+ * In the case that there is a simulator and simulating the code on the new version determines errors, the user can be prompted more obviously than described above to update their code or go back
+ 
+If the editor is not the latest version but the user *has* been redirected from an older version, two possible reasons for this are:
+
+ * The user has dropped a hex file from an older version of the editor than they had loaded, and chosen to be redirected
+ * The user has already moved versions once to a newer version, found fault with that and asked to come back
+
+In these cases, it seems desirable not to prompt the user to move forward.
+
+ {<<I’m up for discussing these workflows, I don’t really want dialogues in front of people, but I also don’t want to move versions under their feet. >>}}
+
+## Versions manifest JSON
+
+This specification relies on a version of the editor being able to know that it is no longer the ‘latest’ version and know the location of newer versions. This could be done by including a versions manifest that lives one level up from the editor.html. This manifest should contain:
+
+ * The current stable version: `”stable” : “2.1”`
+ * A list of all supported versions `”supported”: [ “1.3”, “2.1” ]`
+ * A list of deprecated versions (not sure this is necessary?)
+ 
+ {<< This seems very MVP, might be better to instead have every version listed and a status for each one? >>}
```

Comment:
Well, the idea was actually that the manifest was a level up from the versions, so not included in them, as to do it in them we'd have to update every version at every release of another version which is suboptimal. But maybe we're crossed wires?